financial-services-accelerator/internal-webapps/org.wso2.financial.services.accelerator.demo.backend/src/main/java/org/wso2/financial/services/accelerator/demo/backend/services/AccountService.java

```diff
@@ -0,0 +1,203 @@
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.financial.services.accelerator.demo.backend.services;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.wso2.financial.services.accelerator.demo.backend.BankException;
+
+import java.util.UUID;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+/**
+ * AccountService class.
+ */
+@Path("/accountservice/")
+public class AccountService {
+
+    @SuppressFBWarnings("JAXRS_ENDPOINT")
+    // Suppressed content - Endpoint
+    // Suppression reason - False Positive : This endpoint is a demo endpoint that is not exposed in production
+    // Suppressed warning count - 1
+    @GET
+    @Path("/accounts/{AccountId}")
+    @Produces("application/json; charset=utf-8")
+    public Response getOneAccount(@PathParam("AccountId") String accountId,
+                                  @HeaderParam("x-fapi-interaction-id") String xFapiInteractionId,
+                                  @HeaderParam("Account-Request-Information") String accountRequestInfo)
+            throws BankException {
+
+        String response = "{\n" +
+                "  \"Data\": {\n" +
+                "    \"Account\": [\n" +
+                "      {\n" +
+                "        \"AccountId\": \"" + accountId + "\",\n" +
+                "        \"Status\": \"Enabled\",\n" +
+                "        \"StatusUpdateDateTime\": \"2020-04-16T06:06:06+00:00\",\n" +
+                "        \"Currency\": \"GBP\",\n" +
+                "        \"AccountType\": \"Personal\",\n" +
+                "        \"AccountSubType\": \"CurrentAccount\",\n" +
+                "        \"Nickname\": \"Bills\",\n" +
+                "        \"Account\": [{\n" +
+                "          \"SchemeName\": \"SortCodeAccountNumber\",\n" +
+                "          \"Identification\": \"" + accountId + "\",\n" +
+                "          \"Name\": \"Mr Kevin\",\n" +
+                "          \"SecondaryIdentification\": \"00021\"\n" +
+                "        }]\n" +
+                "      }\n" +
+                "    ]\n" +
+                "  },\n" +
+                "  \"Links\": {\n" +
+                "    \"Self\": \"https://api.alphabank.com/open-banking/v3.0/accounts/" + accountId +
```

Comment:
```suggestion
                "    \"Self\": \"https://api.alphabank.com/open-banking/v4.0/accounts/" + accountId +
```

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/135/commits/a4f614eb3ad84ebf84d43e930eaaedc736898359